src/main/java/com/microsoft/azure/spring/data/documentdb/core/DocumentDbOperations.java

```diff
@@ -21,7 +21,8 @@
     DocumentCollection createCollectionIfNotExists(String collectionName,
                                                    String partitionKeyFieldName,
                                                    Integer requestUnit,
-                                                   IndexingPolicy policy);
+                                                   IndexingPolicy policy,
```

Comment:
Better to refine this API like https://github.com/Microsoft/spring-data-cosmosdb/pull/107 (Another cherry-pick). Because the arguments count from this method is 5 now. We prefer not to add more in future.

Reply:
Wow, this PR is only 22hours old. :D Will look into it later...

Reply:
Yeah. I create that just FYI.